code/modules/medical/computer/cloning.dm

```diff
@@ -447,12 +447,21 @@
 			break
 	return selected_record
 
+/obj/machinery/computer/cloning/process()
+	update_icon()
+
 /obj/machinery/computer/cloning/update_icon()
```

Comment:
Moving it out of process, and only having it update when actually necessary when one of the icon_state condition changes as well as the above will likely be necessary